custom_components/e3dc_rscp/coordinator.py

```diff
@@ -205,6 +254,22 @@ def _process_manual_charge(self, request_data) -> None:
         #     request_data, "EMS_MANUAL_CHARGE_LASTSTART"
         # )[2]
 
+    def _process_powermeters_data(self, powermeters_data) -> None:
+        for powermeter_data in powermeters_data:
+            if powermeter_data["index"] != 0:
+                for powermeter_config in self._e3dcconfig["powermeters"]:
+                    if powermeter_data["index"] == powermeter_config["index"]:
+                        self._mydata[powermeter_config["key"]] = (
+                            powermeter_data["power"]["L1"]
+                            + powermeter_data["power"]["L2"]
+                            + powermeter_data["power"]["L3"]
+                        )
+                        self._mydata[powermeter_config["key"] + "_total"] = (
```

Comment:
see comments on the PR itself, ideally, we try to get this from E3DCs built int history function.

Reply:
See discussions in the PR; After checking the RSCP Excel I am pretty sure the DB just does not provide ready to use power / energy data.